geode-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/string/AbstractSetNXIntegrationTest.java

```diff
@@ -78,7 +87,18 @@ public void testSetNX() {
     assertThat(response3).isEqualTo(0);
   }
 
-  private String randString() {
-    return Long.toHexString(Double.doubleToLongBits(Math.random()));
+  @Test
+  public void testSetNX_whenCalledConcurrently() {
+    Jedis jedis2 = new Jedis("localhost", getPort(), REDIS_CLIENT_TIMEOUT);
```

Comment:
don't forget to close jedis2 at the end of the test

Reply:
Done!